components/admin-tabs/Applicants.tsx

```diff
@@ -89,6 +252,35 @@ const Applicants: React.FC = () => {
     downloadFile(data.data.totalCount, filters, sorter).then(() => setExporting(false));
   };
 
+  const onChange: TableProps<SingleRecordType>['onChange'] = (pagination, filters, sorter) => {
+    setSorter(sorter);
+    setPagination(pagination);
+    setFilters(filters);
+  };
+
+  const components = {
+    body: {
+      row: EditableRow,
+      cell: EditableCell,
+    },
+  };
+
+  columns = columns.map((col) => {
```

Comment:
instead of mutating columns, this would be better to do something like `const columns2 = columns.map(...` just so we avoid re-mutating over and over and over again